client/client/src/EntriesView.tsx

```diff
@@ -0,0 +1,73 @@
+import { readEntries } from './data';
+import { MouseEventHandler, ReactNode } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPencil } from '@fortawesome/free-solid-svg-icons';
+type Props = {
+  entriesView: string;
+  onNewButtonClick: () => void;
+  onEditButtonClick: MouseEventHandler<SVGSVGElement>;
+};
+export default function EntriesView({
+  entriesView,
+  onNewButtonClick,
+  onEditButtonClick,
+}: Props) {
+  const liveEntries: ReactNode[] = [];
+  const localStorageEntries = readEntries();
+  if (localStorageEntries) {
+    localStorageEntries.forEach((element) => {
+      liveEntries.push(
+        <li data-entry-id={element.entryId} key={element.entryId}>
+          <div className="row">
+            <div className="column-half">
+              <img
+                className="input-b-radius form-image"
+                src={element.formURL}></img>
+            </div>
+            <div className="column-half">
+              <div className="row">
+                <div className="column-full d-flex justify-between align-center">
+                  <h3>{element.formTitle}</h3>
+                  <FontAwesomeIcon
+                    cursor="pointer"
+                    icon={faPencil}
+                    onClick={onEditButtonClick}
+                  />
+                </div>
+                <p>{element.formNotes}</p>
+              </div>
+            </div>
+          </div>
+        </li>
+      );
+    });
+  }
```

Comment:
This works, however we don't need to create an empty array to push to (i.e `const liveEntries`) when we take advantage of ES6! This is because the array `map` method returns a new array with the callback function called on every element!

Check out `line 66`'s comment, using `map` is the current industry standard ;)

Here's the React Rendering Lists docs again for a refresher if needed: https://react.dev/learn/rendering-lists#rendering-data-from-arrays